lib/commands/create-project.ts

```diff
@@ -12,6 +13,10 @@ export class ProjectCommandParameter implements ICommandParameter {
 				this.$errors.fail("You must specify <App name> when creating a new project.");
 			}
 
+			if (value.toUpperCase() === "APP") {
```

Comment:
The project may be shared through version control with other people who has Mac OS. I think it is best to be shown always.